documentation/modules/configuring/con-config-openshift-security-providers.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// assembly-security-providers.adoc
+
+[id='con-config-openshift-security-providers-{context}']
+= Applying security context to pods and containers on OpenShift
+
+[role="_abstract"]
+OpenShift uses security context constraints (SCCs) to control permissions for pods.
+Security context constraints (SCCs) are composed of settings and strategies that control the security features a pod has access to.
+
+By default, OpenShift injects security context configuration automatically.
+In most cases, this means you don't need to configure security context for the pods and containers created by the Cluster Operator.  
+Although it is possible to create your own SCCs.
+
+For more information, see the {OpenShiftDocs}. 
```

Comment:
Can we keep this a bit more generic as it is in the blog post? 
* There are / might be other platforms doing this + users can install additional tools to do this
* On OCP, the security context is injected by default, but even that can be disabled etc., so this might not always apply

Reply:
Okay. I changed the title and intro, but used OpenShift as an example. Changed the name of the file.